.jscsrc

```diff
@@ -1,3 +1,4 @@
 {
-  "preset": "google"
+  "preset": "google",
+  "requireCamelCaseOrUpperCaseIdentifiers": "ignoreProperties"
```

Comment:
Why not camel case on properties?  We've been doing that elsewhere with javascript